### PR TITLE
Add wcmatch library (for st builds >=4070)

### PR DESCRIPTION
**Pull request by rwols**
As preparation for a follow-up PR for #1282, this adds the wcmatch library as another dependency.

### CODE REVIEW

File: dependencies.json [hunk 1]
```diff
@@ -1,13 +1,23 @@
 {
     "*": {
-        ">=3124": [
+        "3124 - 4069": [
```
Comment (rwols) [inline dependencies.json:3]: Hmm, I'm unsure if this is the right syntax.

Comment (rwols) [reply dependencies.json:3]: Seems valid https://github.com/wbond/package_control/blob/67ef2e02fef941c01694a52c5dfc3a08f86fbe49/example-dependencies.json#L20-L21

Comment (rchl) [reply dependencies.json:3]: I don't see the point of those version dependant checks anyway as this code base won't work with ST3 or even older ST4 builds.

Comment (rwols) [reply dependencies.json]: Fair point, I've removed the checks.

**Review by rchl — APPROVED** [review]

**Pull request merged**